docs/components/multiselect.md

```diff
@@ -67,33 +100,44 @@ You can specify `disabledTooltipText` property to customize the disabled tooltip
   }, {
     label: 'A long & truncated item',
     value: 'long'
-  }, {
-    label: 'Duck',
-    value: 'duck',
-    group: 'Birds'
-  },{
-    label: 'Salmon',
-    value: 'salmon',
-    group: 'Fish'
-  }, {
-    label: 'Oriole',
-    value: 'oriole',
-    group: 'Birds'
-  }, {
-    label: 'Trout',
-    value: 'trout',
-    group: 'Fish'
-  }]"
+  }, ...]"
 />
 ```
 
+### help
+
+Pass a `string` of help text to be displayed below the multiselect element.
+
+<ClientOnly>
+  <KMultiselect label="Label" help="Help text or other tips." :items="deepClone(defaultItemsUnselect)" />
+</ClientOnly>
+
+```html
+<KMultiselect label="Label" help="Help text or other tips." :items="items" />
+```
+
+### error
+
+A `boolean` to put the component into an error state.
+
+<ClientOnly>
+  <KMultiselect error :items="deepClone(defaultItemsCollapse)" />
+  <br>
+  <KMultiselect error collapsed-context help="Help text can turn into error message." :items="deepClone(defaultItemsCollapse)" />
+</ClientOnly>
+
+```html
+<KMultiselect error :items="items" />
+<KMultiselect error collapsed-context help="Help text can turn into error message." :items="items" />
+```
+
 ### enableItemCreation
 
-`KMultiselect` can offer users the ability to add custom items to the list by typing the item they want to and then clicking the `... (Add new value)` item at the bottom of the list, which will also automatically select it.
+KMultiselect can offer users the ability to add custom items to the list by typing the item they want to and then clicking the `... (Add new value)` item at the bottom of the list, which will also automatically select it.
```

Comment:
<img width="422" alt="Screenshot 2024-01-03 at 9 13 02 PM" src="https://github.com/Kong/kongponents/assets/2229946/2b7b768b-4170-403a-86f8-1240674ab9de">

issue: we need to sanitize strings that are added to prevent injecting HTML and script tags into the DOM